src/crawlee/storage_clients/_redis/_utils.py

```diff
@@ -0,0 +1,22 @@
+from collections.abc import Awaitable
+from pathlib import Path
+from typing import TypeVar, overload
+
+T = TypeVar('T')
+
+
+@overload
+async def await_redis_response(response: Awaitable[T]) -> T: ...
+@overload
+async def await_redis_response(response: T) -> T: ...
+
+
+async def await_redis_response(response: Awaitable[T] | T) -> T:
+    """Solve the problem of ambiguous typing for redis."""
+    return await response if isinstance(response, Awaitable) else response
+
+
+def read_lua_script(file_path: Path) -> str:
```

Comment:
Couldn't this accept just the file name and prepend the path to the `lua_scripts` directory automatically?